# page 1038
std_msgs/Header header

BinaryHeader log_header

#Data
uint32 week
float64 seconds

float64 pitch_rate
float64 roll_rate
float64 yaw_rate

float64 lateral_acceleration
float64 longitudinal_acceleration
float64 vertical_acceleration